source/get-started/connect-to-mongodb.txt

```diff
@@ -0,0 +1,84 @@
+.. _cpp-quick-start-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a project directory
+
+      In your shell, run the following command to create a directory called
+      ``cpp-quickstart`` for this project:
```

Comment:
Q: Should this command be run from the root directory? I am still in the `build` dir at this point.

Reply:
Oh yes you shouldn't be inside mongo-cxx-driver-r3.10.1 for this step, I'll clarify that
```suggestion
      In your shell, run the following command to create a directory called
      ``cpp-quickstart`` for this project:
```